content/plugins/descriptions/aws-lambda.yaml

```diff
@@ -19,7 +19,7 @@ logo:
   # The overall size of the file should be approximately 200 by 200.
   # The image in the file should be approximately 100 by 100 pixels. It needs space around it.
   # The image will be made greyscale by Gatsby Image Sharp.
-  fileSystemPath: './content/assets/logos/aws-lambda/logo.png'
+  fileSystemPath: './content/assets/logos/aws-lambda/logo-lambda.png'
```

Comment:
Same comment as the API Docs. 

![Screenshot 2020-11-04 at 12 30 34](https://user-images.githubusercontent.com/562403/98112147-972bed00-1e99-11eb-908b-533873c59a72.png)

As a general rule, white and transparent should be the only colors used in logos.

If that's not viable, I'm open to changing the design of this page completely so that colored logos work better.

Reply:
I made a preview how it might looks with colored logos: 
![localhost_8000_backstage_plugins (1)](https://user-images.githubusercontent.com/36006588/98128381-2942ee00-1eb8-11eb-8b1c-02ce59c9a6f1.png)

I my opinion it looks better and would be easier to maintain. I also changed sizes of the logos to be more compact. Also I removed visible white pixels from Firebase Functions logo.